packages/vuetify/src/components/VAvatar/VAvatar.sass

```diff
@@ -1,8 +1,8 @@
-@import '../../styles/styles.sass'
+@import './_variables.scss'
 
 .v-avatar
   align-items: center
-  border-radius: 50%
+  border-radius: $avatar-border-radius
```

Comment:
Use it also in skeleton loader's avatar

Reply:
Hi @jacekkarczmarczyk, ty for help! I'm going through framework using a TODO list, will ref your review there!